tox.ini

```diff
@@ -13,6 +13,7 @@ deps =
     -rocrd_models/requirements.txt
     -rocrd_modelfactory/requirements.txt
     -rocrd_validators/requirements.txt
+    -rocrd_network/requirements.txt
```

Comment:
I completely forgot we still have the tox setup. I never use it nowadays, CI handles that. Should we drop it?

Reply:
Not sure what is best here.